code-challenges.js

```diff
@@ -21,12 +31,29 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+const filtAndRemainder =  (arr) => {
+//  filter only numbers using typeof and map them getting only the remainder of the numbers divided by 3.
+  return arr.filter ( value => typeof value === 'number' ).map (value => value % 3)
+  // console.log (filtered)   
+  
+}
 
-
-
+console.log("2.1)", filtAndRemainder(testingArray1))
+console.log("2.2)", filtAndRemainder(testingArray2))
 
 // --------------------3) Create a function that takes in two arrays as arguments returns one array with no duplicate values.
 
 var testingArray3 = [3, 7, "hi", 10, 3, "hello", 4, "hi"]
 var testingArray4 = [7, "hi", 3, 1, "hi", 4, "hello", 4, 7]
 // Expected output: [ 3, 7, "hi", 10, "hello", 4, 1 ]
+
+  const noDup = (arr1, arr2) => {
+    // merging the two arrays in one using .concat 
+    let nodup = arr1.concat(arr2)
+    // filter the new array using hte indexOf built in method.
+    return nodup.filter((value, index) => 
+    // The indexOf() method returns the position of the first occurrence of a specified value in a string.
+    nodup.indexOf(value) === index
+    )
+  }
+    console.log ("3:", noDup(testingArray3,testingArray4))
```

Comment:
good job!! we solved it the exact same way. my only suggestion would be not to name your variable inside the function the same thing as the function itself (even though it was missing the capital D so it still ran successfully- this can end up being confusing later!)